edgePy/io/_io.py

```diff
@@ -6,15 +6,10 @@
 from typing import List, Union
 
 
-__all__ = [
-    'GroupImporter',
-    'DataImporter',
-    'get_dataset_path',
-]
+__all__ = ['GroupImporter', 'DataImporter', 'get_dataset_path', 'get_open_function']
```

Comment:
Tests (including continuous integration provided by TravisCI) aren't passing because you've accidentally added `'get_open_function'` where it doesn't exist. You appear to have merged/rebased on the wrong `master`. Specifically, you've merged in the `HEAD` of the `master` on your fork and not on this remote.

You're a project contributor now so feel free to make branches on this repository in the future, it makes collaboration easier!

Something like:

```
git fetch upstream
git merge upstream/master
```

Reply:
Haha, I thought two conflicts was too little. I'll get that up in a few.